client/app/iplb/configuration/iplb-configuration.controller.js

```diff
@@ -22,46 +41,77 @@ class IpLoadBalancerConfigurationCtrl {
     });
   }
 
-  $onInit() {
-    this.zones.load()
-      .then(() => {
-        this.startPolling();
-      });
+  applyChanges(zone) {
+    this.zones.loading = true;
+    let promise = this.$q.resolve([]);
 
-    this.selectedZones = [];
-  }
+    const zoneData = _.has(this.zones, 'data') ? this.zones.data : this.zones;
 
-  onSelectionChange(selection) {
-    this.selectedZones = selection;
-  }
+    const targets = _.isArray(zone)
+      ? zone
+      : [this.zones.data.find(currentZone => currentZone.id === zone)];
```

Comment:
You can destructure : 
```suggestion
      : [this.zones.data.find(({ id }) => id === zone)];
```

Reply:
Done